app/(admin)/admin/notifications/actions/sendEmailNotification.test.tsx

```diff
@@ -0,0 +1,35 @@
+'use server';
+import { messaging } from "@/api/serverConfig";
+import { sendEmailNotifications } from "./sendEmailNotification";
+
+jest.mock('./sendEmailNotification', () => ({
+    sendEmailNotifications: jest.fn(),
+}))
+
+jest.mock('@/api/serverConfig', () => ({
+    messaging: {
+        createEmail: jest.fn(),
+    },
+}));
+
+describe('SendEmailNotification', () => {
+    it('should send email with provided information', async () => {
+        const content = 'Test';
+        const sendEmailUsers = ['123456', '12345', '1234'];
+        const subject = 'This is a test';
```

Comment:
Instead of declaring these as variables, could we hard code some of these values into the tests as they are only used twice?

Reply:
can you eloborate on this? i don't quite understand

Reply:
Sure, so what I mean is you could do the following:

```
        (sendEmailNotifications as jest.Mock).mockImplementation(async ({content, sendEmailUsers, subject}) => {
            await (messaging.createEmail as jest.Mock)('1234567890', 'This is a test' , 'Test', [], sendEmailUsers);
        })
```

and remove `content` and `subject` as variables since we're only using them in this test twice

Reply:
i wanted to test that the variables are being passed through correctly. this function is given props from the frontend and it's a backend function. I wanted to make sure that it receives the props correctly and passes it through the function.